### PR TITLE
Update id-field.asciidoc

### PR DESCRIPTION
**Pull request by markwalkom**
Adding a note around the size limit for `_id`

### CODE REVIEW

**Review by cbuescher — COMMENTED** [review]
@markwalkom thanks, the change looks good, but there were errors in the docs build. I merged in the current 7.1 branch again to see if this was a temporary hickup, if not would you mind checking what might be wrong with the formatting of the note?

**Comment by javanna** [comment]
retest this please

**Comment by javanna** [comment]
@elasticmachine update branch

**Comment by javanna** [comment]
heya @jrodewig may I ask you to check what is making the tests fail here? Is it something to do with the newly added note?

File: docs/reference/mapping/fields/id-field.asciidoc [hunk 1]
```diff
@@ -39,3 +39,8 @@ but doing so is discouraged as it requires to load a lot of data in memory. In
 case sorting or aggregating on the `_id` field is required, it is advised to
 duplicate the content of the `_id` field in another field that has `doc_values`
 enabled.
+
+
+[NOTE]
+==================================================
+`_id` is limited to 512 bytes in size and larger values will be rejected.
```
Comment (jrodewig) [inline docs/reference/mapping/fields/id-field.asciidoc]: The syntax here is causing docbldesx to fail. Just need to add a closing delimiter to the [NOTE] block.

```suggestion
`_id` is limited to 512 bytes in size and larger values will be rejected.
==================================================
```

Comment (javanna) [reply docs/reference/mapping/fields/id-field.asciidoc]: great, thanks a lot for checking and suggesting a fix!!!

**Pull request merged**